data/planet.lua

```diff
@@ -307,24 +329,26 @@ local planets = {
         },
     },
     {
-        type = "planet",
-        name = "luna",
-        icon = "__real-starry-universe__/graphics/luna.png",
+        type = "planet", -- You should be able to land on asteroids in the asteroid belt.
```

Comment:
*Small asteroids, if you've ever played SE and went to orbit - This is what I'm thinking (I've played SE and been to orbit)

The change of planet name to "asteroid-belt-outer" is because I moved luna up towards earth.